config/environments/development.rb

```diff
@@ -40,6 +40,8 @@ module OptimizerWrapper
   WEEKDAYS = %i[mon tue wed thu fri sat sun].freeze
   DEMO = Wrappers::Demo.new(tmp_dir: TMP_DIR)
   VROOM = Wrappers::Vroom.new(tmp_dir: TMP_DIR, threads: 1)
+  RELATION_ZIP_CLUSTER_CAN_HANDLE = %i[minimum_day_lapse maximum_day_lapse minimum_duration_lapse maximum_duration_lapse force_first never_first force_end
```

Comment:
zip cluster cannot ensure that force_first, never_first or force_end are respected if multiple services with different position preference are within the same zip cluster